cds/modules/legacy/admin.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Admin model views for CDSMigrationLegacyRecord."""
+
+import html
+import json
+
+from flask_admin.contrib.sqla import ModelView
+from flask_wtf import FlaskForm
+from invenio_admin.filters import FilterConverter
+from invenio_pidstore.models import PersistentIdentifier
+from markupsafe import Markup
+
+from .models import CDSMigrationLegacyRecord
+
+
+class CDSMigrationLegacyRecordModelView(ModelView):
```

Comment:
Is it worth adding a new admin view, given that it is dead?

Reply:
We said it is handy for checking if something is wrong with a migrated record... we will still use invenio-admin for a while in Videos :)